src/Classes/Utils/Utils.ts

```diff
@@ -27,6 +27,17 @@ export class Utils {
         } : null;
     }
 
+    /**
+     * Generate a random RGBA color
+     */
+    public static getRandomColorRGBA(min?: number) {
+        return {
+            r: Math.floor(Math.random() * 255 + (min || 0)),
+            g: Math.floor(Math.random() * 255 + (min || 0)),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'g' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)